CardData.py

```diff
@@ -182,12 +185,14 @@ def create_card_json(gwent_data_helper, patch):
         card['categoryIds'] = []
         categoriesSum = int(template.find('Categories').find('e0').attrib['V']);
         # XML Card category is the sum of all categories of the card
-        for category in sorted(CATEGORIES, reverse=True):
-            if categoriesSum - category >= 0:
-                categoriesSum -= category
-                card['categoryIds'].append(CATEGORIES[category])
-            if categoriesSum == 0:
-                break
+        for category in range(MAX_CATEGORIES, 1, -1):
```

Comment:
No need for the `MAX_CATEGORIES` constant, just put the `max(CATEGORIES)` in this line

e.g.
`for category in range(max(CATEGORIES), 1, -1):`

Reply:
If you think so. Personally I like to not having the script to recalculate a value that is a const in the end, for each iteration.

Reply:
Good point. Keep the constant, but rename it to `LAST_CATEGORY`.